docs/components/renderer.md

```diff
@@ -32,6 +32,7 @@ The `renderer` system configures a scene's
 | maxCanvasWidth          | Maximum canvas width. Uses the size multiplied by device pixel ratio. Does not limit canvas width if set to -1.                                | 1920            |
 | maxCanvasHeight         | Maximum canvas height. Behaves the same as maxCanvasWidth.                      | 1920          |
 | logarithmicDepthBuffer  | Whether to use a logarithmic depth buffer.                                      | auto          |
+| precision  |       Shader Precision                                | high          |
```

Comment:
should describe it more here, what it does, including the options. I'd almost just shove the whole paragraph from below into here, or at least link it.